lib/src/android_alarm.dart

```diff
@@ -67,6 +67,17 @@ class AndroidAlarm {
       throw AlarmException('Isolate error: $e');
     }
 
+    if (dateTime.difference(DateTime.now()).inSeconds <= 1) {
+      await playAlarm(id, {
+        "assetAudioPath": assetAudioPath,
+        "loopAudio": loopAudio,
+        "vibrate": vibrate,
+        "fadeDuration": fadeDuration.toInt(),
+        "enableNotificationOnKill": enableNotificationOnKill,
```

Comment:
We don't need to pass the arguments `vibrate` and `enableNotificationOnKill` right ?

Reply:
Any update ? Otherwise I can do it myself if you want.

Reply:
Oops, didn't notice your comment. 
I am on a trip right now, so I can't do it now.
I will arrive in three days and fix it. It is an easy fix you could do it yourself if you want.

Reply:
It's your PR so I can't modify it directly, but no worries I'll wait !

Reply:
Thanks!